exercises/parallel-letter-frequency/parallel_letter_frequency_test.go

```diff
@@ -44,3 +44,25 @@ Gave proof through the night that our flag was still there;
 O say does that star-spangled banner yet wave,
 O'er the land of the free and the home of the brave?`
 )
+
+func BenchmarkSequentialFrequency(b *testing.B) {
+	b.StopTimer()
+	b.StartTimer()
+
+	for i := 0; i < b.N; i++ {
+		Frequency(euro + dutch + us)
+	}
+
+	b.StopTimer()
```

Comment:
I have to come clean - I copy-pasted that from another test. If you remove the Timer calls, it appears to make no difference at all.

Not too sure what comes next here, but looks like I should be able to "Add more commits by pushing to the master branch on roundand/xgo" so I'll try that.